core/frontend/src/components/vehiclesetup/MotorDetection.vue

```diff
@@ -0,0 +1,237 @@
+<template>
+  <v-dialog
+    v-if="is_sub"
+    v-model="dialog"
+    max-width="1000px"
+    @input="status = statuses.IDLE"
+  >
+    <template #activator="{ on, attrs }">
+      <v-btn
+        color="primary"
+        dark
+        v-bind="attrs"
+        v-on="on"
+      >
+        Detect Reversed Motors
+      </v-btn>
+    </template>
+
+    <v-card class="pa-5" style="overflow-x: hidden;">
+      <v-card-title class="text-h5 justify-center">
+        Automatic Motor Reversal Detection
+      </v-card-title>
+      <v-sheet v-if="display_status" class="elevation-1 pa-3 ma-8">
+        <StatusTextWatcher :filter="/thrust|Thrust|Motor/" @message="updateStatus" />
+      </v-sheet>
+      <v-row v-if="status == statuses.IDLE">
+        <v-col rows="3" class="justify-center">
+          <ThemedSVG src="/public/img/icons/motordetection.svg" />
+        </v-col>
+        <v-col cols="8">
+          <v-card-text>
+            <p>
+              Tests whether any of the motors are wired to spin backwards,
+              and automatically reverses their control signals as relevant.
+            </p>
+            <p>
+              The test relies on the <b>motors being correctly connected</b>
+              according to the selected vehicle frame, and
+              spins each motor individually to check whether it generates thrust in the expected direction.
+              The vehicle should be in calm water and far from walls and objects.
+              The autopilot must be in manual mode to be able to run the test.
```

Comment:
This line should be removed, as commented here: https://github.com/bluerobotics/BlueOS/pull/2760/files#r1663308816